PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -145,171 +145,8 @@ intptr_t CALLBACK DebugInfoDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 	{
 		case WM_INITDIALOG:
 		{
-			NppParameters& nppParam = NppParameters::getInstance();
-

```

Comment:
Lines 148 & (152 thru' 312) used for constructing the Debug Info text has been moved out as is from `WM_INITDIALOG` into the new function `refreshDebugInfo()`. This new function will be called from `DebugInfoDlg::doDialog()`. This refactoring allows for the Debug Info text to be refreshed when the command line string changes.